bigquery/tests/system.py

```diff
@@ -564,7 +603,12 @@ def _write_csv_to_storage(self, bucket_name, blob_name, header_row,
                 blob.upload_from_file(csv_read, content_type='text/csv')
 
         self.to_delete.insert(0, blob)
+        return 'gs://{}/{}'.format(bucket_name, blob_name)
 
+    def _write_avro_to_storage(self, bucket_name, blob_name, avro_file):
```

Comment:
There doesn't seem to be anything Avro-specific about this function. Should you be setting a content type as is done in the CSV case?

Per https://issues.apache.org/jira/browse/AVRO-488 the MIME-type `application/x-avro-binary` may be appropriate.